packages/ui/src/ui/store/selectors/global/index.ts

```diff
@@ -195,6 +195,14 @@ export const getGlobalDefaultPoolTreeName = (state: RootState) =>
     state.global.poolTreeDefault || 'physical';
 
 export const getGlobalShowLoginDialog = (state: RootState) => {
+    if (state.global.authWay === 'passwd') {
+        return (
+            getConfigData().allowLoginDialog &&
+            state.global.ytAuthCluster &&
+            (!state.global.login || state.global.showLoginDialog)
+        );
+    }
```

Comment:
Do we really need the `if`-statement? The condition bellow returns true for the inner condition, doesn't it?